sdks/java/nexmark/src/test/java/org/apache/beam/sdk/nexmark/PerfsToBigQueryTest.java

```diff
@@ -111,6 +114,7 @@ public void testSavePerfsToBigQuery() throws IOException, InterruptedException {
     expectedRows.add(row1);
     TableRow row2 =
         new TableRow()
+            .set("timestamp", start.getMillis())
```

Comment:
`TableRow.set()` accepts `Object` so I'm a little bit afraid of runtime errors in case it doesn't understand long. :) Is BigQuery able to convert long to TIMESTAMP when we pass `start.getMillis()` (long value) in a table row?

Reply:
TableRow.set() takes an Object indeed. But I think it is using autoboxing from long to Long. As I cannot pass the Joda Instant that would never be understood by BigQuery, the long is the best option IMHO as the write-then-read test passes.

Reply:
Ok. As long as long gets interpreted by BigQuery correctly we should be fine with this.